pkg/acrn-kernel/Dockerfile

```diff
@@ -0,0 +1,99 @@
+FROM alpine:3.10 as kernel-build
+#linuxkit/alpine:f2f4db272c910d136380781a97e475013fabda8b AS kernel-build
+
+RUN apk update
+
+RUN apk add \
```

Comment:
can you please add --no-cache here and remove the RUN apk update line?

Reply:
@rvs agree. I will hold on this patch and  let's discuss about why there are so many patches in the acrn-kernel and how we can handle those big diffs and what our kernel upstreaming plan is.